observation/qos/evm_interpreter.go

```diff
@@ -91,6 +95,19 @@ func (i *EVMObservationInterpreter) GetServiceID() (string, bool) {
 	return serviceID, true
 }
 
+// GetRequestOrigin returns the Origin of the request:
+// Organic, i.e. User requests.
+// Synthetic, i.e. requests build by the QoS sytem to get observations on endpoints.
```

Comment:
Can you add A TODO to standardize the language on organic vs synthetic requests across docs/comments/code somewhere?

Feel free to assign it to me.